src/windows/index.rst

```diff
@@ -29,7 +29,7 @@ To install and use DeepStack GPU version on your Windows machine, follow the ste
 
 - Ensure your machine has an NVIDIA GPU
 
-- Download and Install `CUDA 10.1 from NVIDIA <https://developer.nvidia.com/cuda-10.1-download-archive-base>`_ 
+- Download and Install `CUDA 11.0 from NVIDIA <https://developer.nvidia.com/cuda-11.0-download-archive>`_

```

Comment:
We no longer need this change. Pull the latest changes from master and merge it into this branch.